timeless/roles/views.py

```diff
@@ -33,6 +36,9 @@ def create():
 
 @bp.route("/edit/<int:id>", methods=("GET", "POST"))
 def edit(id):
+    table = Role.query.get(id)
+    if not table:
+        return abort(HTTPStatus.NOT_FOUND)
```

Comment:
@ekondrashev this ``if`` should go in the next one, right? We should only try to read the table if it's a ``POST``.

Reply:
@amihaiemil right, fixed, thx!